numba_dpex/tests/kernel_tests/test_math_functions.py

```diff
@@ -24,31 +35,44 @@ def unary_op(request):
     np.float64,
 ]
 
+N = 2048
+
 
 @pytest.fixture(params=list_of_dtypes)
 def input_arrays(request):
     # The size of input and out arrays to be used
-    N = 2048
     a = np.array(np.random.random(N), request.param)
     b = np.array(np.random.random(N), request.param)
-    return a, b
+    c = np.zeros(N, dtype=np.int64)
+    return a, b, c
 
 
 @pytest.mark.parametrize("filter_str", filter_strings)
 def test_binary_ops(filter_str, unary_op, input_arrays):
-    a, actual = input_arrays
+    a, actual, actual_value_types = input_arrays
+
+    if unary_op == "ceil" or unary_op == "floor":
+        a = a * 10.0
+
     uop = getattr(math, unary_op)
     np_uop = getattr(np, unary_op)
 
     @dpex.kernel
-    def f(a, b):
+    def f(a, b, c):
         i = dpex.get_global_id(0)
-        b[i] = uop(a[i])
+        k = uop(a[i])
+        if type(k) is int or type(k) is np.int32 or type(k) is np.int64:
+            c[i] = 3
```

Comment:
This test actually doesn't verify if `ceil` and `floor` works properly. It only checks type.
Please:
* Leave original `test_binary_ops` as is
* Create separate test to test type
* Add test for integer data?

Also we must be sure, that `ceil` and `floor` functions actually tested on data with fraction.